### ISSUE TITLE
Negative numbers and line 101

### ISSUE THREAD

**Issue by kdauzickas** [bobthecow/genghis#29]
Trying to save an object which contains a field with a negative number gives an UnaryExpression error on line 18 (even if the object is 3 lines long). 
E.g. { foo: -1} will be impossible to save

Another issue: when editing long documents, only 101 lines are drawn. The others can be seen only after clicking/scrolling below the rendered content.

**Comment by bobthecow** [comment]
Do you mind opening another bug for line 101 issue? I'm testing a fix for the UnaryExpression error right now.
